packages/gatsby-plugin-sharp/README.md

```diff
@@ -24,8 +24,7 @@ of PNGs then it can significantly reduce build times.
 
 ## How to use
 
-```javascript
-// In your gatsby-config.js
```

Comment:
This only applies to README files: GitHub doesn't render the title thing, so I'd keep this in here. It's one of the only cases where we can be more specific for GitHub (unlike linking, which we keep relative to gatsbyjs.org).